src/blog/2023/07/how-to-build-a-opc-client-dashboard-in-node-red.md

```diff
@@ -0,0 +1,233 @@
+```
+
+We have two values to write, a boolean value corresponding to the node object `MySwitch`, and an integer value corresponding to the object `MyLevel`. Therefore, we will use a toggle switch to toggle the `MySwitch`, and a slider to modify `MyLevel`.  
+
+There’s no need to modify the toggle switch properties, other than giving it a name.  The slider needs to have the range modified to match the range of the level, which is 0-100%.
+
+![level-range.png](./images/opc-ua-3/level-range.png)
+For the `OpcUa-Item` nodes, copy the `NodeId` corresponding to each device,
+
+![copy-node-id.png](./images/opc-ua-3/copy-node-id.png)
+and paste it into `OpcUa-Item` node.  You must also ensure the data-type matches with the value you’re writing to.
+
+![opcua-item.png](./images/opc-ua-3/opcua-item.png)
+The `Opc-Ua-Client` needs to have an endpoint and the action changed to `WRITE`.  
+
+![client-node.png](./images/opc-ua-3/client-node.png)
+The process is the same for `MySwitch` and `MyLevel`, the only difference being what `NodeId` is referenced in the  `OpcUa-Item` node.  
+
+When deployed, you can confirm values are being written to from the client to the server from the dashboard.
+
+<video width="560" height="315" controls>
+  <source src="https://website-data.s3.eu-west-1.amazonaws.com/opc-write.mp4" type="video/mp4">
+</video>
+## Read Alarms & Events from OPC UA Server Using OpcUa-Event and Opc-Ua-Client Nodes
```

Comment:
```suggestion

## Read Alarms & Events from OPC UA Server Using OpcUa-Event and Opc-Ua-Client Nodes
```

Reply:
This title is not showing up correctly. I am not sure why?

Reply:
looks like you caught it.  just needed a space added.  kinda strange.